src/App.jsx

```diff
@@ -20,13 +20,16 @@ import AdminPage from '../src/pages/AdminPage';
 import AccountPage from './pages/AccountPage';
 import SectionPage from './pages/SectionPage';
 import MonitorLogPage from './pages/MonitorLogPage';
+import MonitorLogEditPage from './pages/MonitorLogEditPage';
 import Footer from './components/Footer/Footer';
 import Navbar from './components/Navbar/Navbar';
 import PeoplePage from './pages/PeoplePage';
 import Species from './pages/Species';
+import NewQuestionModal from './components/NewQuestionModal';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'NewQuestionModal' is defined but never used.